Release Notes/Release Notes v2.11.4.md

```diff
@@ -0,0 +1,19 @@
+# .NET Driver Version 2.11.4 Release Notes
```

Comment:
I think you need to create a new PR, since now we have too many unrelated changes